drivers/i2s/i2s_nrfx.c

```diff
@@ -113,6 +113,11 @@ static void find_suitable_clock(const struct i2s_nrfx_drv_cfg *drv_cfg,
 			uint32_t mck_factor =
 				(uint32_t)((requested_mck * 1048576ULL) /
 					   (src_freq + requested_mck / 2));
+
+			/* skip cases when mck_factor is too big for dividing */
+			if (mck_factor > 1048576) {
```

Comment:
How about making this magic number a define? Then it could be used here and in lines 114, 121.

Reply:
I wanted to avoid that as I would also need to include a macro to generate the "ULL" version of the constant; please let me know if you have suggestion how to do this in a way that does not clutter the code

Reply:
suggestion commited